apps/home/templates/apps_home/blocks/quote_block.html

```diff
@@ -7,15 +7,15 @@
                 {{ self.text }}
                     <div>
                         {% if value.link %}
-                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more' %}&#8594;</a>
+                        <a class="quote-ds-block__link" href="{{ value.link.url }}">{% trans 'Read more ' %}<i class="fas fa-arrow-right" aria-hidden="true"></i></a>
```

Comment:
One last thing: There shouldn't be a blank at the end of the translation. Did you add that for the space between the "Read more" and the arrow? Then try to add it between the trans tag and the `<i...`

Reply:
Ok, will remember this. thanks!